### ISSUE TITLE
Could you release the result on validation set?

### ISSUE THREAD

**Issue by Odaimoko** [Microsoft/human-pose-estimation.pytorch#56]
Since this is a more convenient way to compare the results.

**Comment by Odaimoko** [comment]
Sorry. I mean the validation results on posetrack 2017 dataset (for task 1/2/3). There are only results from the test server in the paper right?

**Comment by leoxiaobin** [comment]
Our paper also report the results on posetrack 2018 val dataset.

**Comment by Odaimoko** [comment]
Ah yeah. Sorry for the bother. Thanks a lot.
